### PR TITLE
Note on using the `oc adm` commands on first master

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1432978

### CODE REVIEW

**Comment by mburke5678** [comment]
@sferich888 The `oadm` command has been deprecated since you files the original BZ/ Does the same restrictions apply to the `oc adm` commands?

**Comment by mburke5678** [comment]
@sferich888 PTAL

_When running `oc adm` commands, you should run them only from only the first master listed in the Ansible host inventory file, by default `/etc/ansible/hosts`._

_For example, if you execute certificate commands `oc adm ca`, to create a certificate on the one master, then run the same (or similar) command on a second master  the serial numbers in the certificates might not be be unique (because {product-title} uses a flat file for the serial numbers)._

**Comment by mburke5678** [comment]
@xingxingxia PTAL

**Review by sdodson — APPROVED** [review]

**Comment by xingxingxia** [comment]
FYI, https://bugzilla.redhat.com/show_bug.cgi?id=1432978#c10 , comment from QE of Master sub team who knows CA related background

File: getting_started/configure_openshift.adoc [hunk 1]
```diff
@@ -84,6 +84,12 @@ everything.
 ----
 oc adm policy add-cluster-role-to-user cluster-admin admin
 ----
++
+// tag::ocadm-note[]
+When running `oc adm` commands, you should run them only from 
+the first master listed in the Ansible host inventory file,
+by default `/etc/ansible/hosts`.
```
Comment (adellape) [inline getting_started/configure_openshift.adoc]: Needs filename markup, here and throughout.

**Comment by mburke5678** [comment]
@adellape Changed `/etc/ansible/hosts` to *_/etc/ansible/hosts_*.

**Pull request merged**